src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Entity/InstitutionConfigurationOptions.php

```diff
@@ -71,10 +71,9 @@ public function jsonSerialize()
     {
         return
             [
-                $this->institution->getInstitution() => [
-                    'use_ra_locations'             => $this->useRaLocationsOption,
-                    'show_raa_contact_information' => $this->showRaaContactInformationOption,
-                ],
+                'institution'                  => $this->institution,
```

Comment:
Note that, without a change to the ApiBundle's InstitutionConfigurationOptionsController, `GET /management/institution-configuration/options/dev.organisation.example` will now yield something like this:
```
GET /management/institution-configuration/options/dev.organisation.example

{
  "institution": "dev.organisation.example",
  "use_ra_locations": false,
  "show_raa_contact_information": true
}
```